kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -9,6 +9,8 @@ const router = require('kolibri.coreVue.router');
 const coreActions = require('kolibri.coreVue.vuex.actions');
 const ConditionalPromise = require('kolibri.lib.conditionalPromise');
 const samePageCheckGenerator = require('kolibri.coreVue.vuex.actions').samePageCheckGenerator;
+const getDefaultChannelId = require('kolibri.coreVue.vuex.getters').getDefaultChannelId;
```

Comment:
While we're at it, we can probably move all the channel state to the core state. (e.g. the channel list and current channel ID).

thoughts?